docs/tutorials/deploy-grafana-dashboard.md

```diff
@@ -1,28 +1,46 @@
-# Deploy Grafana Dashboard
+# Access Tracee Metrics in Prometheus and Grafana
+
+Tracee exposes a metrics endpoint. 
+This makes it possible to monitor Tracee like other cloud native workloads.
+
+In this tutorial, we will showcase how to access Tracee metrics through Prometheus and Grafana running through docker containers.
+The tutorial can easily adapted to Kubernetes workloads with the configuration provided.
+
+## Tutorial Overview
 
 Grafana is a visualization tools for exported metrics and logs, most commonly
-used alongside prometheus.
+used alongside Prometheus.
+
+Since version 0.7.0, tracee exports useful runtime metrics to Prometheus.
+
+By using Grafana and the new metrics from tracee, we can deploy a simple
+dashboard which tracks the tracee instance performance and outputs.
 
-Since version 0.7.0, tracee exports useful runtime metrics to prometheus.
+There are two options for accessing Tracee metrics:
+* Running the Tracee Docker Container Image -- Shown in this tutorial
+* Running the Tracee Helm Chart -- Detailed as part of the [Promtail-Tracee](./promtail.md) tutorial
```

Comment:
![image](https://github.com/aquasecurity/tracee/assets/3372117/48f4f30c-a8ef-44f5-9c66-38153e525a7c)


```suggestion
There are two options for accessing Tracee metrics:

- Running the Tracee Docker Container Image -- Shown in this tutorial
- Running the Tracee Helm Chart -- Detailed as part of the [Promtail-Tracee](./promtail.md) tutorial
```

Reply:
@AnaisUrlichs waiting for your input here to merge or not.

Reply:
oh sorry, looking now